extension/content.js

```diff
@@ -13,12 +13,20 @@ function linkifyBranchRefs() {
 		let username = parts.eq(0).text();
 		let branch = parts.eq(1).text();
 
+		// forked repos can have their name changed; grab it from first commit in PR
+		const forkName = document.querySelectorAll('.commit-id')[0].href.split('/')[4];
```

Comment:
Btw, if you want the first element, you could just use `querySelector`.

Here though, it would be better to just do:

```js
const forkName = $('.commit-id').attr('href').split('/')[4];
```